Dockerfile

```diff
@@ -1,25 +1,16 @@
-FROM golang:1.9
+FROM golang:1.9-alpine
 
+RUN apk add --no-cache git libcap shadow
+RUN useradd -ms /bin/bash freegeoip
 COPY cmd/freegeoip/public /var/www
-
 ADD . /go/src/github.com/apilayer/freegeoip
+WORKDIR /go/src/github.com/apilayer/freegeoip/cmd/freegeoip
 RUN \
-	cd /go/src/github.com/apilayer/freegeoip/cmd/freegeoip && \
-	go get -d && go install && \
-	apt-get update && apt-get install -y libcap2-bin && \
-	setcap cap_net_bind_service=+ep /go/bin/freegeoip && \
-	apt-get clean && rm -rf /var/lib/apt/lists/* && \
-	useradd -ms /bin/bash freegeoip
+      go get -d && \
+      go install && \
+      setcap cap_net_bind_service=+ep /go/bin/freegeoip
 
-USER freegeoip
 ENTRYPOINT ["/go/bin/freegeoip"]
+USER freegeoip
 
 EXPOSE 8080
-
```

Comment:
I deleted these comments because I don't think they were bringing much value, however I can put them back due to that being outside the scope of this ticket

Reply:
@justgage there is a great Dockerfile best practices article here.  I thought I would pass it along.
https://docs.docker.com/develop/develop-images/dockerfile_best-practices/

Following these recommendations, specifically for [RUN](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#run), it will create a more readable Dockerfile:
```
FROM golang:1.9-alpine

RUN apk add --no-cache \
    git \
    libcap \
    shadow \
 && addgroup -g 1000 -S freegeoip \
 && adduser -u 1000 -S freegeoip -G freegeoip

COPY cmd/freegeoip/public /var/www
COPY . /go/src/github.com/apilayer/freegeoip

WORKDIR /go/src/github.com/apilayer/freegeoip/cmd/freegeoip

RUN go get -d \
 && go install \
 && setcap cap_net_bind_service=+ep /go/bin/freegeoip

ENTRYPOINT ["/go/bin/freegeoip"]
USER freegeoip

EXPOSE 8080
```
Notice that the ADD was changed to a COPY as per the recommendation here on [transparency](https://docs.docker.com/develop/develop-images/dockerfile_best-practices/#add-or-copy).